Network/Public/Include/Network/Primitives.hpp

```diff
@@ -41,6 +41,44 @@ struct ChannelInfo
     }
 };
 
+/**
+ * @brief The ChannelLeaveInfo struct
+ * @details Channel Leave Info struct for information about channels
+ *          contain user ID and channel Name.
+ */
+struct ChannelLeaveInfo
```

Comment:
And like in 'Channel deleting', is really need to create new struct? If you just want other names of struct just use typedef (or better using = condition).